src/java/fr/paris/lutece/plugins/extend/service/extender/ResourceExtenderService.java

```diff
@@ -1,565 +1,588 @@
+        List<ResourceExtenderDTO> listResources = findByFilter( filter );
+
+        if ( ( listResources != null ) && ( listResources.size( ) == 1 ) )
+        {
+            ResourceExtenderDTO resourceExtender = listResources.get( 0 );
+            resourceExtender.setName( getExtendableResourceName( resourceExtender ) );
+
+            return resourceExtender;
+        }
+
+        return null;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public ResourceExtenderDTO findWildCardResourceExtender( String strExtenderType, String strExtendableResourceType )
+    {
+        ResourceExtenderDTOFilter filter = new ResourceExtenderDTOFilter( strExtenderType, ResourceExtenderDTOFilter.WILDCARD_ID_RESOURCE,
+                strExtendableResourceType );
+        filter.setWideSearch( false );
+
+        List<ResourceExtenderDTO> listResources = findByFilter( filter );
+
+        if ( ( listResources != null ) && ( listResources.size( ) == 1 ) )
+        {
+            ResourceExtenderDTO resourceExtender = listResources.get( 0 );
+            resourceExtender.setName( getExtendableResourceName( resourceExtender ) );
+
+            return resourceExtender;
+        }
+
+        return null;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public ResourceExtenderDTO findResourceExtenderIncludingWildcard( String strExtenderType, String strIdExtendableResource, String strExtendableResourceType )
+    {
+        String strKey = new StringBuilder( "ResourceExtenderDTO_" ).append( strExtenderType ).append( '_' ).append( strIdExtendableResource ).append( '_' )
+                .append( strExtendableResourceType ).toString( );
+
+        ResourceExtenderDTO found = (ResourceExtenderDTO) _extenderCache.get( strKey );
+
+        if ( found == null )
+        {
+            ResourceExtenderDTOFilter filter = new ResourceExtenderDTOFilter( strExtenderType, strIdExtendableResource, strExtendableResourceType );
+            filter.setWideSearch( false );
+            filter.setIncludeWildcardResource( true );
+
+            List<ResourceExtenderDTO> listResources = findByFilter( filter );
+
+            if ( listResources != null )
+            {
+                if ( listResources.size( ) == 1 )
+                {
+                    found = listResources.get( 0 );
+                }
+                else if ( listResources.size( ) == 2 )
+                {
+                    if ( listResources.get( 0 ).getIdExtendableResource( ).equals( strIdExtendableResource ) )
+                    {
+                        found = listResources.get( 0 );
+                    }
+                    else
+                    {
+                        found = listResources.get( 1 );
+                    }
+                }
+                else
+                {
+                    AppLogService.error( "More than 2 ResourceExtenderDTO found for {0}, {1}, {2}.", strExtenderType, strIdExtendableResource, strExtendableResourceType );
```

Comment:
Does the replacement of the parameters work?
use: 
AppLogService.error( "More than 2 ResourceExtenderDTO found for {}, {}, {}.", strExtenderType, strIdExtendableResource, strExtendableResourceType );